plugin/src/main/java/com/owen1212055/biomevisuals/Main.java

```diff
@@ -105,7 +105,7 @@ public void onRegistrySend(final @NotNull RegistrySendEvent event) {
             return;
         }
 
-        for (final var keyedOverride : keyedOverrides) {
+        for (var keyedOverride : keyedOverrides) {
```

Comment:
You didn't actually remove the var. 😢

Reply:
My bad, I thought you wanted me to remove the `final` qualifier only. I'll get rid of `var` now :smile: